src/c++/CMakeLists.txt

```diff
@@ -89,6 +89,12 @@ if(TRITON_ENABLE_TESTS OR TRITON_ENABLE_PERF_ANALYZER)
 endif()
 FetchContent_MakeAvailable(repo-common)
 
+if(TRITON_ENABLE_TESTS)
```

Comment:
`cc_client_test.cc` now includes `http_client.cc` to access code in there to test. Code in there relies on `triton_json.h`, and this cmake change allows the test executable to access `triton_json.h` for compilation.